src/tests/unit/services/email/email-provider-factory.test.ts

```
@@ -0,0 +1,148 @@
+/**
+ * @deprecated EmailProviderFactory is deprecated in favor of dependency injection pattern.
+ * Use email.dependencies.ts for new implementations.
+ * This test is maintained for backward compatibility.
+ */
+describe('EmailProviderFactory (Legacy)', () => {
+  beforeEach(() => {
+    // Reset the singleton instance before each test
+    EmailProviderFactory.resetInstance()
+    
+    // Clear any environment variable overrides
+    delete process.env.NODE_ENV
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)